File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/ride/RideSummaryPayloadResponse.java
package com.lattis.ellipse.data.network.model.response.ride;

import com.google.gson.annotations.SerializedName;

/**
 * Created by ssd3 on 4/11/17.
 */

public class RideSummaryPayloadResponse {

    @SerializedName("trip")
    RideSummaryDataResponse rideSummaryDataResponse;

    RideSummaryDataResponse getRideSummaryDataResponse(){
        return rideSummaryDataResponse;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/ride/RideSummaryPayloadResponse.kt
package com.lattis.data.entity.response.ride

import com.google.gson.annotations.SerializedName
import com.lattis.domain.models.DockHub

/**
 * Created by ssd3 on 4/11/17.
 */

class RideSummaryPayloadResponse {

    @SerializedName("trip")
    internal var rideSummaryDataResponse: RideSummaryDataResponse? = null

    @SerializedName("hub")
    var dockHub:DockHub?=null
}
